README.md

```diff
@@ -73,16 +73,34 @@ See below for a closer examination of this process, and see `docs/images` for sc
+
+- `securedrop_salt`, which contains all files required during the provisioning (orchestration) stage. Qubes uses SaltStack internally for VM provisionining and configuration management (see https://www.qubes-os.org/doc/salt/). The contents of `securedrop_salt` are provisioned to end users via the securedrop-workstatoin-dom0-config RPM.
+- `files`, which contain scripts and configuration files placed in dom0 before provisioning, such as systemd units, [RPC policy files](https://www.qubes-os.org/news/2020/06/22/new-qrexec-policy-system/#policy-files), and Python scripts. The contents of `files` are provisioned to end users via the securedrop-workstatoin-dom0-config RPM.
+- the "pre-flight updater" components (`launcher`, `sdw_updater`, `sdw_notify`, `sdw_util`), which comprise a wrapper around the Qubes Updater, and which serve to enforce a fully-patched system. The contents of these directories are provisioned to end users via the securedrop-workstation-dom0-config RPM. The updater updates all TemplateVMs relevant to the SecureDrop Workstation prior to use, and the the `sdw-notify.py` script reminds the user to update the system if they have not done so recently.
+
+This repo also contains the following developer-facing components:
+- `rpm-build`, containing the components required for building the `securedrop-workstation-dom0-config` RPM that is ultimately published to our [yum repository](https://yum.securedrop.org).
+- `scripts`, which are developer-facing linting and build-related scripts.
 - `Makefile` is used with the `make` command on `dom0` to build the Qubes/SecureDrop installation, and also contains some development and testing features.
+
+SecureDrop Workstation has a companion repository, SecureDrop Client, that contains component code for all of the packages we ship
+in individual VMs once they have been provisioned.
 - The [SecureDrop Client](https://github.com/freedomofpress/securedrop-client) is installed in `sd-app` and will be used to access the SecureDrop server *Journalist Interface* via the SecureDrop proxy.
 - The [SecureDrop Proxy](https://github.com/freedomofpress/securedrop-client/tree/main/proxy#readme) is installed in `sd-proxy` to communicate to the SecureDrop server *Journalist Interface* via `sd-whonix`.
-- Within `sd-app`, the *SecureDrop Client* will open all submissions in the `sd-viewer` disposable VM.
+- [SecureDrop Export](https://github.com/freedomofpress/securedrop-client/tree/main/export#readme) is installed in `sd-devices` and is used to manage printing and exporting files.
+- There are other components as well: the *SecureDrop Client* opens all submissions in the `sd-viewer` disposable VM; and a logging VM, `sd-log`, is provisioned to capture logs locally from various parts of the system; and a [Whonix](https://www.whonix.org/wiki/Homepage) VM, `sd-whonix`, is provisioned with instance-specific information required to access the authenticated onion service used by journalists.
```

Comment:
You could just enumerate these components separately in the same way as the previous two, they have their own (meta)packages etc even if they're not as complex.